resources/php_ini.rb

```diff
@@ -0,0 +1,25 @@
+resource_name :php_ini
+
+default_action :add
+
+property :mode, String, default: '0644'
+property :options, Hash, default: {}, required: true
+
+action :add do
+  directory "/etc/php.d #{new_resource.name}" do
+    path '/etc/php.d'
+  end
```

Comment:
Why do you have a special `directory` resource name, instead of just using the `path` option as the name?

Reply:
See one of my previous comments above. Otherwise you have a resource duplication issue.